src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+$stars_size: 16px;
+
 body {
   margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's advised to use `0` instead of `0px` for the `margin` property. Additionally, the font size for the body is typically set in `px` or `rem` units for better control over text scaling. Consider setting a base font size in pixels and using `rem` for font sizes in other selectors to maintain relative sizing.